segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/Tier2Config.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.storage.impl;
+
+import io.pravega.common.util.ConfigBuilder;
+import io.pravega.common.util.ConfigurationException;
+import io.pravega.common.util.Property;
+import io.pravega.common.util.TypedProperties;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Configuration for the Tier 2.
+ */
+@Slf4j
+public class Tier2Config {
```

Comment:
The PR leaves one approach to implement #1290. I recommend that we revisit this part when we design and implement that feature.
These configs are already used in platfrom PR. I would rather not change these unless there is a strong reason as it will cause one more interation.
Do you have a strong reason for changing this?